src/Package/Impl/DataInspect/REnvironmentProvider.cs

```diff
@@ -16,6 +17,10 @@ internal class REnvironmentProvider : IREnvironmentProvider, IDisposable {
         public REnvironmentProvider(IRSession session) {
             _rSession = session;
             _rSession.Mutated += RSession_Mutated;
+
+            IdleTimeAction.Create(() => {
+                GetEnvironmentAsync().SilenceException<Exception>().DoNotWait();
```

Comment:
`SilenceException<Exception>()` is almost always wrong. Ideally, this should only silence those exceptions that are actually expected during normal operation - for stuff that uses `IRSession`, this is typically `MessageTransportException`, and `OperationCanceledException`, but `DoNoWait` automatically silences the latter. If it really must be a catch-all, it should also log them, otherwise we'll have weird behavior reported by users with no way to diagnose.